applications/dashboard/controllers/api/UsersApiController.php

```diff
@@ -598,6 +598,35 @@ public function post_requestPassword(array $body) {
         $this->validateModel($this->userModel, true);
     }
 
+    /**
+     * @param $id
+     * @param array $body
+     * @return array $result The response body
```

Comment:
I don't think this tag is in the right format. [Checkout its docs](http://docs.phpdoc.org/references/phpdoc/tags/return.html).